examples/src/main/java/com/squareup/moshi/recipes/ReadJsonListKt.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.moshi.recipes
+
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import com.squareup.moshi.adapter
+import com.squareup.moshi.recipes.models.Card
+
+internal class ReadJsonListKt {
+
+  //language=JSON
+  private val jsonString = """
+    [{"rank": "4",
+       "suit": "CLUBS"
+     },
+     {"rank": "A",
+      "suit": "HEARTS"
+      },
+     {"rank": "J",
+      "suit": "SPADES"
+     }]
+  """.trimIndent()
+
+  @OptIn(ExperimentalStdlibApi::class)
+  fun readJsonList() {
+    val moshi = Moshi.Builder().build()
```

Comment:
Why the newBuilder() call vs just setting it up in this builder?

Reply:
Simplified the builder. Thanks!